addons/website_blog/views/website_blog_components.xml

```diff
@@ -23,6 +23,7 @@
             </ul>
             <!-- 'Search Box' -->
             <t t-call="website.website_search_box_input">
+                <!-- <t t-set="_form_classes" t-valuef="ms-auto"/> -->
```

Comment:
@Brieuc-brd I am not sure that ms-auto was useful in your solution, can you confirm?

Reply:
Indeed, this margin isn't necessary, I can't remember why I added it 🤔.